get-deps

```diff
@@ -1,100 +1,132 @@
-#!/bin/bash
+#!/bin/sh
 
-if ! bash ./ci/check-rust-version.sh ; then
+test_cmd() {
+  command -v $1 >/dev/null
+}
+
+if test_cmd 'bash' && test_cmd 'rustc' && ! bash ./ci/check-rust-version.sh ; then
```

Comment:
It's important that they have rustc and that they have an appropriate version of rustc.
This change effectively disables these preconditions on systems that don't have bash.
Can you expand this PR to remove the `bash` dep from `ci/check-rust-version.sh` as well?

Reply:
I've skipped that since it uses Bash arrays. I have an idea how to rewrite it in posix shell, just need to test it.